app/views/api/v2/common/show_hidden.json.rabl

```diff
@@ -0,0 +1,7 @@
+object @parameter
+
+if params["show_hidden"] == "true" && @object.present? && @object.editable_by_user?
```

Comment:
is the `@object.present?` check still needed? wouldn't a missing object cause a failure anyways when attempting to access the attribute?